### PR TITLE
toolbar and header Section of Main-Mobile are created in this pull request

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
Summary :
This pull request has the following changes.
- the toolbar is added.
- header is added
- index file is added
- linters are added
- Styling on the main mobile is done.
- project description is added
Overall First 2 Sections of this project are done in this branch with complete styling.

### CODE REVIEW

File: .github/.hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
```
Comment (emmiiorji) [inline .github/.hintrc]: You have made a good effort at setting up linters to run on your local environment. Note that the `.hintrc` and `.stylelintrc.json` files are supposed to be in the root directory of the project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    <!-- Header -->
+    <header class="header">
+      <a href="#" class="logo">
+        <h3>My Logo</h3>
+      </a>
+      <button class="menu_btn">
+        <div class="menu">
+          <img src="images/menu.png" alt="" />
+        </div>
+      </button>
```
Comment (emmiiorji) [inline index.html]: Kindly use a `nav` tag for the hamburger. This is more semantically correct as it helps the user navigate sections of the site more easily.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  width: 24px;
+  height: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+  background-image: url('images/mobile_bg.png');
```
Comment (emmiiorji) [inline style.css]: Kindly crosscheck that the url of the background image is correct. Currently, there seems not to be a `mobile_bg.png` file in the images folder.

Your design:
image

Figma Design:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        </div>
+        <div class="info">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="social_media"><a href="#">LET’S CONNECT</a></div>
```
Comment (emmiiorji) [inline index.html]: Kindly adjust the color and design of the LET'S CONNECT button to match that of the design template. It has no underline in the Figma design.

Your design:
image

Figma deisgn:
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        <div class="title">
+          <h1>I’m Mudassir Glad to see you!</h1>
+        </div>
+        <div class="info">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
```
Comment (emmiiorji) [inline index.html:41]: I recommend that you ensure that the headline text, intro text, LET'S CONNECT and social icons are vertically aligned as shown by the design template.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Rana-Mudassir, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Almost there

## Good Highlights✔

- Linter checks are passing
- Good commit messages
- README is professional
- Adherence to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (Rana-Mudassir) [reply index.html:41]: All issues are resolved.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile-Main</title>
```
Comment (Yazino12) [inline index.html:7]: - **[OPTIONAL]** Kindly consider using a title that represents what your website is all about 🆗. This is what your users will see in their browser tabs and it'll give them an idea of what your site stands for 💪🏽.

**Review by Yazino12 — APPROVED** [review]
## Hi @Rana-Mudassir🖐🏾,

You've done well implementing the changes requested by a previous reviewer ✨, there is nothing else left to say but it's time to merge it 🚀🚀🚀.

### To highlight:
- Professional readme file ✔️ 
- Linters are passing ✔️ 
- Clean code ✔
- Descriptive PR summary ✔
- Good commit messages ✔

# STATUS: APPROVED ✅✅✅
DONE
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**